doc/developer/repoproviders.rst

```diff
@@ -0,0 +1,60 @@
+.. _providers-section:
+
+====================
+Repository Providers
+====================
+
+Repository Providers (or RepoProviders) are
+locations where git repositories are stored (e.g.,
```

Comment:
```suggestion
locations where repositories are stored (e.g.,
```
The Zenodo or FigShare providers don't (need to) give us something that is a Git repository (contains a `.git` subdirectory). I think calling the thing we start from a "repository" is a good compromise between using something specific (c.f. "directory" is super vague I think, even though it is strictly true) and trying to move away from the idea that it has to be a git repository.

Reply:
good point, totally agree